linkis-commons/linkis-common/src/test/java/org/apache/linkis/common/utils/ByteTimeUtilsTest.java

```diff
@@ -17,35 +17,156 @@
 
 package org.apache.linkis.common.utils;
 
-import static org.junit.jupiter.api.Assertions.*;
+import java.util.function.Function;
 
-import org.apache.linkis.common.exception.LinkisSecurityException;
+import com.google.common.collect.ImmutableMap;
 import org.junit.jupiter.api.Assertions;
 import org.junit.jupiter.api.Test;
+
 class ByteTimeUtilsTest {
 
-    @Test
-    void byteStringAsBytes() {
-    }
+  private static final ImmutableMap<String, Function<String, Long>> opFunction =
+      ImmutableMap.<String, Function<String, Long>>builder()
+          .put("byteStringAsBytes", tar -> ByteTimeUtils.byteStringAsBytes(tar))
+          .put("byteStringAsKb", tar -> ByteTimeUtils.byteStringAsKb(tar))
+          .put("byteStringAsMb", tar -> ByteTimeUtils.byteStringAsMb(tar))
+          .put("byteStringAsGb", tar -> ByteTimeUtils.byteStringAsGb(tar))
+          .build();
 
-    @Test
-    void byteStringAsKb() {
-    }
+  @Test
+  void byteStringAsBytes() {
+    convertToByte.forEach(
+        (k, v) -> Assertions.assertEquals(opFunction.get("byteStringAsBytes").apply(k), v));
+    Assertions.assertThrows(
+        IllegalArgumentException.class, () -> opFunction.get("byteStringAsBytes").apply("1A"));
+  }
 
-    @Test
-    void byteStringAsMb() {
-    }
+  @Test
+  void byteStringAsKb() {
+    convertToKB.forEach(
+        (k, v) -> Assertions.assertEquals(opFunction.get("byteStringAsKb").apply(k), v));
+    Assertions.assertThrows(
+        IllegalArgumentException.class, () -> opFunction.get("byteStringAsKb").apply("1a"));
+  }
 
-    @Test
-    void byteStringAsGb() {
-        Long res = ByteTimeUtils.byteStringAsGb("1G");
-        Assertions.assertEquals(res,1);
+  @Test
+  void byteStringAsMb() {
+    convertToMB.forEach(
+        (k, v) -> Assertions.assertEquals(opFunction.get("byteStringAsMb").apply(k), v));
+    Assertions.assertThrows(
+        IllegalArgumentException.class, () -> opFunction.get("byteStringAsMb").apply("1c"));
+  }
 
-        Assertions.assertThrows(
-            NullPointerException.class,
-            () -> {
-                ByteTimeUtils.byteStringAsGb("512");
-            });
+  @Test
+  void byteStringAsGb() {
+    convertToGB.forEach(
+        (k, v) -> Assertions.assertEquals(opFunction.get("byteStringAsGb").apply(k), v));
+    Assertions.assertThrows(
+        IllegalArgumentException.class, () -> opFunction.get("byteStringAsGb").apply("1C"));
+  }
 
-    }
-}
\ No newline at end of file
+  private static final ImmutableMap<String, Long> convertToByte =
```

Comment:
about static code 
best to put it in front of the test

Reply:
done